test/functional/relative-time/format.js

```diff
@@ -0,0 +1,109 @@
+define([
```

Comment:
We need `test/functional/relative-time/format-relative-time.js` to test Globalize's `.formatRelativeTime()` API and `test/functional/relative-time/relative-time-formatter.js` to test Globalize's `.relativeTimeFormatter()` API.

Given `.formatRelativeTime()` is basically an alias for `.relativeTimeFormatter()()`, it's only important to test the validations for the former (there's no need to duplicate other tests that will be present for `.relativeTimeFormatter()`). Please, just let me know in any question.